astroplan/constraints.py

```diff
@@ -122,22 +164,24 @@ class AltitudeConstraint(Constraint):
     Constrain the altitude of the target.
 
     .. note::
-        This will misbehave if you try to constrain negative altitudes, as
+        This can misbehave if you try to constrain negative altitudes, as
         the `~astropy.coordinates.AltAz` frame tends to mishandle negative
-        altitudes.
-    """
-    def __init__(self, min=None, max=None):
-        """
-        Parameters
-        ----------
-        min : `~astropy.units.Quantity` or `None`
-            Minimum altitude of the target (inclusive). `None` indicates no limit.
 
-        max : `~astropy.units.Quantity` or `None`
-            Maximum altitude of the target (inclusive). `None` indicates no limit.
-        """
+
+    Parameters
+    ----------
+    min : `~astropy.units.Quantity` or `None`
+        Minimum altitude of the target. `None` indicates no limit.
+    max : `~astropy.units.Quantity` or `None`
+        Maximum altitude of the target. `None` indicates no limit.
+    boolean_constraint : bool
+        If True, the constraint is treated as a boolean (True for within the
+        limits and False for outside).  If False, the constraint returns a
+        float on [0, 1], where 0 is the min altitude and 1 is the max.
+    """
+    def __init__(self, min=None, max=None, boolean_constraint=True):
         if min is None:
-            self.min = 0*u.deg
+            self.min = -90*u.deg
```

Comment:
What motivates this change? I don't think anyone observes below the horizon, so I don't see a motivation for the minimum altitude to be <0.